### PR TITLE
Feature/292680 hcsamples

### PR DESCRIPTION
**Pull request by joBueche**
add some samples for statistics in MaterialManager

### CODE REVIEW

**Comment by joBueche** [comment]
@HOMAG-Boris-Wehrle , @HOMAG-Robin-Katzenberger  Please have a look at the changes.
I recreated the docu from XML for the material contracts, but it created a lot more than my 2 changed classes.
We must decide, if we always automatically generate the markdow with "xmldoc2md" or we edit manually.

**Comment by HOMAG-Robin-Katzenberger** [comment]
> @HOMAG-Boris-Wehrle , @HOMAG-Robin-Katzenberger Please have a look at the changes. I recreated the docu from XML for the material contracts, but it created a lot more than my 2 changed classes. We must decide, if we always automatically generate the markdow with "xmldoc2md" or we edit manually.

I think the automatic generation is kinda unnecessary, it is a nice to have but only if it is 100% correct. I don´t think that anyone will take a look at it. For me it would make more sense to have good documentation besides the things that are already obvious by taking a look into the class / interface etc. itself.

File: Applications/MaterialManager/Documentation/SourceCode/homagconnect.materialmanager.client.materialmanagerclientprocessing.md
Comment (HOMAG-Robin-Katzenberger) [inline Applications/MaterialManager/Documentation/SourceCode/homagconnect.materialmanager.client.materialmanagerclientprocessing.md:1]: I think you can also remove them for intelliDivide, as they also have them inside their source code documentation

Comment (joBueche) [reply Applications/MaterialManager/Documentation/SourceCode/homagconnect.materialmanager.client.materialmanagerclientprocessing.md:1]: Done

**Review by HOMAG-Robin-Katzenberger — APPROVED** [review]

**Review by HOMAG-Robin-Katzenberger — APPROVED** [review]

**Review by HOMAG-Robin-Katzenberger — APPROVED** [review]

**Pull request merged**